code/modules/mob/living/emote.dm

```diff
@@ -334,6 +354,56 @@
 	message = "sighs."
 	emote_type = EMOTE_AUDIBLE
 
+/datum/emote/living/sigh/get_sound(mob/living/user)
+	if(iscarbon(user))
+		if(user.gender == MALE)
+			return 'sound/emotes/male/male_sigh.ogg'
+		return 'sound/emotes/female/female_sigh.ogg'
+	return
+
+/datum/emote/living/sniff
+	vary = TRUE
+
+/datum/emote/living/sniff/get_sound(mob/living/user)
+	if(iscarbon(user))
+		if(user.gender == MALE)
+			return 'sound/emotes/male/male_sniff.ogg'
+		return 'sound/emotes/female/female_sniff.ogg'
+	return
+
+
+/datum/emote/living/snore
+	vary = TRUE
+	sound = 'sound/emotes/snore.ogg'
+
+/datum/emote/living/burp
+	vary = TRUE
+
+/datum/emote/living/burp/get_sound(mob/living/user)
+	if(iscarbon(user))
+		if(user.gender == MALE)
+			return 'sound/emotes/male/burp_m.ogg'
+		return 'sound/emotes/female/burp_f.ogg'
+	return
```

Comment:
These three are still semi-modular since they're overriding existing emotes rather than actually editing them.
The original emote definitions are here, if you could move these into them:
https://github.com/pariahstation/Pariah-Station/blob/b083e79b170e80b134e9858d274274b1eae61619/code/modules/mob/living/emote.dm#L358-L362
https://github.com/pariahstation/Pariah-Station/blob/b083e79b170e80b134e9858d274274b1eae61619/code/modules/mob/living/emote.dm#L364-L370
https://github.com/pariahstation/Pariah-Station/blob/b083e79b170e80b134e9858d274274b1eae61619/code/modules/mob/living/emote.dm#L44-L48

Reply:
I love modularisation